cli_reference/basic_cli_operations.adoc

```diff
@@ -202,10 +202,29 @@ $ oc project <project_name>
 Return a list of objects for the specified xref:object-types[object type]. If
 the optional `<object_name>` is included in the request, then the list of
 results is filtered by that value.
+
 ----
 $ oc get <object_type> [<object_name>]
 ----
 
+The `oc get <object_type>` provides the `-o`, `--output` options to modify the output format.
+----
+$ oc get <object_type> [<object_name>]-o|--output=json|yaml|wide|custom-columns=...|custom-columns-file=...|go-template=...|go-template-file=...|jsonpath=...|jsonpath-file=...]
+----
+The output format can be a simple JSON or YAML, or an extensible format like  http://kubernetes.io/docs/user-guide/kubectl-overview/#custom-columns[custom columns], http://golang.org/pkg/text/template/#pkg-overview[golang template], and  http://kubernetes.io/docs/user-guide/jsonpath[jsonpath].
+
+For example:
+
+The following command lists the name of the pods running in a specific project.
```

Comment:
I think these can be merged, so:

"For example, the following command lists the name of the pods running in a specific project:"

Notice the colon after "project" too.

Reply:
@bfallonf : Thanks for your feedback